### PR TITLE
Try and fix youtube

### PR DESCRIPTION
**Pull request by mattheu**
Move unwrap logic to new method so we can reuse the code.

### CODE REVIEW

File: simple-facebook-instant-articles.php [hunk 1]
```diff
@@ -174,8 +174,8 @@ public function customise_feed_query( $query ) {
 		// Customise FB IA feed query.
 		if ( $query->is_main_query() && $query->is_feed( $feed_slug ) ) {
 
-			$query->set( 'posts_per_rss', 25 );
-			$query->set( 'orderby', 'modified' );
+			$num_posts = intval( apply_filters( 'simple_fb_posts_per_rss', get_option( 'posts_per_rss', 10 ) ) );
+			$query->set( 'posts_per_rss', $num_posts );
```
Comment (dashaluna) [inline simple-facebook-instant-articles.php:179]: @mattheu grrr ... not sure why this change got reverted here? What's the best thing to do now? Have a separate PR fixing it - or revert this current PR?

Comment (mattheu) [inline simple-facebook-instant-articles.php:179]: Oh will have been a merge conflict... good catch. I'll just push a revert commit straight to master
